src/types/feeBreakdown.ts

```diff
@@ -21,10 +23,14 @@ export type ReductionOrExemption =
 export interface PassportFeeFields {
   "application.fee.calculated": number;
   "application.fee.payable": number;
-  "application.fee.payable.includesVAT": boolean;
   "application.fee.reduction.alternative": boolean;
   "application.fee.reduction.parishCouncil": boolean;
   "application.fee.reduction.sports": boolean;
   "application.fee.exemption.disability": boolean;
   "application.fee.exemption.resubmission": boolean;
+  "application.fee.serviceCharge"?: number;
+  "application.fee.serviceCharge.VAT"?: number;
+  "application.fee.fastTrack"?: number;
+  "application.fee.fastTrack.VAT"?: number;
+  // `${string}.VAT`?: string;
```

Comment:
```suggestion
  [key: `${string}.VAT`]: number | undefined;
```

I think it's just the `[key: x]` syntax missing here 👍

Reply:
Yep - just for clarity it's needed there also. The tests just cover the helper function `sumVAT()` - if we were to add other `.vat` variables into the actual passport (any input into `getFeeBreakdown()` it would fail as these would be stripped away as additional keys).

PR https://github.com/theopensystemslab/planx-core/pull/695 will resolve this 👍